packages/go_router/README.md

```diff
@@ -36,7 +36,15 @@ See the API documentation for details on the following topics:
 - [Named routes](https://pub.dev/documentation/go_router/latest/topics/Named%20routes-topic.html)
 - [Error handling](https://pub.dev/documentation/go_router/latest/topics/Error%20handling-topic.html)
 
-## Migration guides
+## Breaking Change
+Only changes that requires manual fix are considered breaking changes. Changes that can be fixed
+through `dart fix` are not considered breaking change in this package.
+
+Every breaking change increases major version. If your application is broken without a major version
+upgrade, run `dart fix --apply` to fix your application. If it does fix the breakages, please file
```

Comment:
If the dart fix doesn't work, that means we accidentally break the api without knowing. Otherwise, we would have increase major version.

It will be no different with or without this new policy

Reply:
> If the dart fix doesn't work, that means we accidentally break the api without knowing.

No, in the scenario I'm describing `dart fix` will never work:
- Application A depends on package B.
- Package B depends on `go_router`.

When you break go_router, the developer of **A** has a broken build the next time they sync their packages. Running `dart fix` in A **doesn't do anything**, because the usage is in package B, not in the application A. And the author of A cannot run `dart fix` on package B because it's a published package they don't own.